client/my-sites/themes/themes-selection.jsx

```diff
@@ -119,31 +123,32 @@ const ThemesSelection = React.createClass( {
 	},
 
 	render() {
-		const site = this.props.selectedSite;
+		const { selectedSite: site } = this.props;
```

Comment:
😄  For a brief moment, I was considering declaring a bunch of other locals here. I ended up deciding against it, but I couldn't bring myself to un-destructure this again.